src/data/store/fetch-all.js

```diff
@@ -0,0 +1,33 @@
+import QueryBuilder from '../query-builder'
+
+const MAX_PAGE_SIZE = 100 // max allowed value
+
+async function fetchAll(whereClause) {
+  whereClause = whereClause || new QueryBuilder()
+
+  if (!(whereClause instanceof QueryBuilder)) {
```

Comment:
it looks weird, perhaps it would be better to rename `whereClause` into `queryBuilder`?

Reply:
Done